variables.tf

```diff
@@ -351,6 +357,12 @@ variable "repositories" {
       topics      = ["mage-os", "magento", "ecommerce", "magento2", "adobecommerce"]
     }
 
+    mageos-data-migration-tool = {
```

Comment:
Do we need a mage-os-specific repository for this project? Do we plan to work on this? Do we release it?

Reply:
@Vinai -- can the package be released to repo.mage-os.org without having our own version of it?

Reply:
Yes it can